rpadutils/rpadutils.py

```diff
@@ -21,12 +21,16 @@
 from redbot.core.bot import Red
 from redbot.core.utils.chat_formatting import box, pagify
 
+RPADCOG = None
 
 class RpadUtils(commands.Cog):
     def __init__(self, bot: Red, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.bot = bot
 
+        global RPADCOG
+        RPADCOG = self
```

Comment:
Ugh, do you absolutely have to do this? This is the worst pattern I've ever used. I don't recommend copying it.

Reply:
i'm not a fan either but it's really effective.  how would you recommend accessing the bot outside of a cog?

Reply:
i'm not a fan either but it's really effective.  how would you recommend accessing the bot outside of a cog?

Reply:
If you absolutely have to access the bot outside a cog, fine, but that's super rare (the only time I had to do this was as part of an auth decorator).

Pass the bot as an argument into the function that needs it; you're probably calling from a cog which has access to the bot.

Reply:
You marked this resolved but this global is still in here. I see the usage has been removed though.